code/tailer/datastripe.q

```diff
@@ -27,33 +27,55 @@ modaccess:{[accesstab]};
     modaccess[.ds.access];
 
     // call the savedown function
-    .ds.savealltablesoverperiod[.ds.td;nextp;lasttime];
+    .ds.savealltablesoverperiod[.ds.td;lasttime];
     .lg.o[`reload;"Kept ",string[.ds.periodstokeep]," period",$[.ds.periodstokeep>1;"s";""]," of data from : ",", " sv string[t]];
     
     // update the access table on disk
-    atab:get ` sv(.ds.td;.proc.procname;`access);
+    accesspath: ` sv(.ds.td;.proc.procname;`$ string .wdb.currentpartition;`access);
```

Comment:
Should we be trying to move away from .wdb namesapce for currentpartition and other variables? It is fine for wdb functions I think (especially until Conor finishes the wdb-tailer split, but for variables it is a bit confusing I think.

Reply:
IMO it depends if this variable is unique to the tailer or if it's present & referenced in wdb code - if the latter, my opinion would be it should stay in `.wdb` namespace rather than duplicating

Reply:
The .wdb.currentpartition is used in the same way in the wdb endofday function as I have used it in the tailer's endofday